### ISSUE TITLE
microsoft meeting

### ISSUE THREAD

**Issue by xxali24** [kyleneideck/BackgroundMusic#481]
I have installed it and it is amazing but one disadvantage in my opinion is that if I am in a teams meeting and I want to lower the meetings audio I can't  there is no option to lower the meetings audio, only the app.
which is frustrating because this was the main feature I wanted from this program.

thank you

**Comment by kyleneideck** [comment]
Have you checked that the "Background Music" device is selected in the Teams settings? And have you tried looking in Background Music's "More Apps" menu for apps like "Microsoft Teams Helper"?

I haven't tried it myself, but I'm wondering if maybe the volume slider labelled "Microsoft Teams" in the main Background Music menu controls might control the app's audio (UI sounds?), but you need to use one of the "Microsoft Teams Helper" sliders in More Apps for meetings. It would be nice to fix it so the main slider works for both, but no one's gotten around to that yet.

There are some other issues related to Teams that might help too:

#268 #423 #441 #419 #282

**Comment by b3z** [comment]
also if you run it in browser. In my case the safari output is not the Safari App but one of the "apple.webkit..." ones. (In this case the 3rd one which is not zero in the right list.)

![image](https://user-images.githubusercontent.com/47346598/116524929-43461200-a8d8-11eb-93a4-563db1a6fcc0.png)

**Comment by EccentricHyphus37** [comment]
As @kyleneideck said — as well as plenty of others in the linked issues, the main `Microsoft Teams.app` (which shows up in the main BackgroundMusic menu) ***does not* handle meeting audio**, which is **delegated** to one of the *many instances* of the [*ridiculously large number of*] `Microsoft Teams Helper *.app`s. So toggling the volume for one of these in the *More Apps* drawer should do the trick.
And I **can confirm** that **it does**.

**Caveats:**
Hovering over a menu entry with ellipses (…) doesn’t present the full name of the app, and all that is visible is “Microsoft Teams H…”.
Fortunately for me, in **Big Sur, only one** of these many **Helper** apps actually play sound, so finding it and then controlling Teams meetings’ audio is easy. However, I’ve seen screenshots from some of the linked issues on **older versions** of macOS where there are **multiple** instances, and this could demand some trial and error.

**Comment by xxali24** [comment]
Thank you so much it worked

I really appreciate the work that you have put into this.

**Comment by DennisGaida** [comment]
>  So toggling the volume for one of these in the _More Apps_ drawer should do the trick.
> And I **can confirm** that **it does**.

Can also confirm that this works. It would be awesome if we could "tag" those apps from More Apps so they are on the main screen? I would actually love to be able to configure which apps are displayed and hide many of the default apps.

**Comment by EccentricHyphus37** [comment]
Agreed! I’d love the **option to switch** from having the main menu display _only_ the **visible** apps to displaying the running ones of a customised list of mine, or something similar — like a quick access menu of pinned apps.
